Additions/UIView-Debugging.m

```diff
@@ -91,6 +101,36 @@ - (void)printControlState {
     ctrl.highlighted ? printf(" (highlighted)") : printf(" (not highlighted)");
 }
 
+- (void)printDatePickerState {
```

Comment:
Does the current date picker value get printed as part of the normal `printAccessibilityElementsWithIndentation:` call?

Reply:
It does look like it's getting printed currently. In the issue [here](https://github.com/kif-framework/KIF/issues/1146) you can see what is currently printed.

Reply:
Ok, so the `UIAccessibilityPickerComponent` of the UIDatePicker itself will still be printed it looks like? SGTM

Reply:
Good callout on that I checked again and I had to add a line for stopping after the `_UIDatePickerView` 

I checked and this is what the DatePicker portion of the print hierarchy looks like:

```
UIDatePicker (enabled) (not selected) (not highlighted) (date range: no minimum - no maximum) (mode: UIDatePickerModeDate) (minute interval: 1)
|	|	|	|	_UIDatePickerView
|	|	|	|	UIAccessibilityPickerComponent, label: (null), value: December, traits: selected, adjustable
|	|	|	|	UIAccessibilityPickerComponent, label: (null), value: 31, traits: selected, adjustable
|	|	|	|	UIAccessibilityPickerComponent, label: (null), value: 2030, traits: selected, adjustable
```

Thinking about it do you think we'd want to have the other labels accessibility in here? Specifically the lines like:
```
UITableViewCellAccessibilityElement, label: (null), traits: none
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIPickerTableViewWrapperCell
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIAccessibilityElementMockView, label: 00 minutes, traits: none
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UITableViewCellContentView
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIView
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIDatePickerContentView, label: 00 minutes
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UILabel, label: 00
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIPickerTableViewWrapperCell
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIAccessibilityElementMockView, label: 05 minutes, traits: none
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UITableViewCellContentView
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIView
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIDatePickerContentView, label: 05 minutes
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UILabel, label: 05
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIPickerTableViewWrapperCell
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIAccessibilityElementMockView, label: 10 minutes, traits: none
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UITableViewCellContentView
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIView
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIDatePickerContentView, label: 10 minutes
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UILabel, label: 10
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	_UIScrollViewScrollIndicator, label: Horizontal scroll bar, 1 page
|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	|	UIView
```

It might be too much but i think it could be doable if we had special cases in `printAccessibilityElementsWithIndentation` for views under `UIDatePIcker`. 

This could also be filed under an improvement and we could merge this.

Reply:
The rest seems too verbose to me, as I think it can essentially be inferred from the data already available, right?

Reply:
Yep i think so. Going to merge as is